app/src/ui/merge-branch/merge.tsx

```diff
@@ -143,7 +149,7 @@ export class Merge extends React.Component<IMergeProps, IMergeState> {
               Merge into <strong>{currentBranch ? currentBranch.name : ''}</strong>
             </Button>
           </ButtonGroup>
-          {mergeInfo}
+          {this.renderMergeInfo(invalidBranch)}
```

Comment:
Yea I noticed that. So should we change it to `{invalidBranch ? null : this.renderMergeInfo}` or leave it